backend/sublet/serializers.py

```diff
@@ -91,6 +92,19 @@ class Meta:
             # but gets on sublets will include ids/urls for images
         ]
 
+    def validate_title(self, value):
+        if self.contains_profanity(value):
+            raise serializers.ValidationError("The title contains inappropriate language.")
+        return value
+
+    def validate_description(self, value):
+        if self.contains_profanity(value):
+            raise serializers.ValidationError("The description contains inappropriate language.")
+        return value
+
+    def contains_profanity(self, text):
+        return predict([text])[0]
```

Comment:
to confirm, do we need to lowercase the text or anything? or is this package already handle this

Reply:
it's handled